packages/vuetify/src/composables/position.ts

```diff
@@ -0,0 +1,52 @@
+// Utilities
+import type { PropType } from 'vue'
+import { computed } from 'vue'
+import { convertToUnit } from '@/util/helpers'
+import propsFactory from '@/util/propsFactory'
+
+// Types
+export interface PositionProps {
+  bottom?: boolean | string
+  left?: boolean | string
+  position?: 'static' | 'relative' | 'fixed' | 'absolute' | 'sticky'
+  right?: boolean | string
+  top?: boolean | string
+}
+
+const positionValues = ['static', 'relative', 'fixed', 'absolute', 'sticky'] as const
+
+// Composables
+export const makePositionProps = propsFactory({
+  bottom: [Boolean, String],
+  left: [Boolean, String],
+  position: {
+    type: String as PropType<Partial<typeof positionValues[number]>>,
+    validator: /* istanbul ignore next */ (v: any) => positionValues.includes(v),
```

Comment:
Is this worth it just to artificially increase coverage?

Reply:
Okay Mr. OCD about classes as a string.